blosc/schunk.c

```diff
@@ -141,19 +143,46 @@ blosc2_schunk* blosc2_schunk_new(const blosc2_storage storage) {
   // ...and update internal properties
   update_schunk_properties(schunk);
 
-  if (storage.sequential) {
+  if (!storage.sequential && storage.path != NULL){
+    char* urlpath;
+    char last_char = storage.path[strlen(storage.path) - 1];
+    if (last_char == '\\' || last_char == '/') {
+      urlpath = malloc(strlen(storage.path));
+      strncpy(urlpath, storage.path, strlen(storage.path) - 1);
+      urlpath[strlen(storage.path) - 1] = '\0';
+    }
+    else {
+      urlpath = malloc(strlen(storage.path) + 1);
+      strcpy(urlpath, storage.path);
+    }
+    //Create directory
+    if (mkdir(urlpath,0777) == -1) {
```

Comment:
Missing space after comma.

Reply:
Fixed in 4607ab374e512f47bc7a64117e9af8907c18fc81